src/microbe_stage/MicrobeAI.cs

```diff
@@ -38,13 +39,17 @@ public class MicrobeAI
     [JsonProperty]
     private float pursuitThreshold;
 
+    private Dictionary<Compound, float> previouslyAbsorbedCompounds;
```

Comment:
I just found out that this doesn't NEED a [JsonProperty] annotation for serialization, but to avoid confusing a future dev I would like to see these all having the annotation, or having them all removed (which maybe leads to more complications?) if you want to earmark spring cleaning into this PR.

Reply:
I wondered about the annotation, but I must confess I don't know these at all in C#. I thus left it out, in order not to break anything. Should I put a [JsonProperty] or a [JsonIgnore]?

From what I get it seems to affect saves (made as JSON files), therefore a [JsonIgnore] could make the gradient wrong for one execution at file load. Yet this is likely not a major issue as overall movement depends on many executions. I'm really unsure about this one.

Reply:
In this case, you will want JsonProperty.

Reply:
I did so, and tested to be sure. It seems to all work well with saves!

Reply:
As this is a private *field* this does need the JsonProperty annotation to be saved. Only public properties with both public set and get, are automatically assumed to be JsonProperty, all other stuff that needs saving, needs that explicit property.  Also conversely, if there's a public property that shouldn't be saved, it needs the JsonIgnore property.